<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>100</height>
   </rect>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>212</red>
      <green>208</green>
      <blue>200</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>234</red>
      <green>229</green>
      <blue>220</blue>
     </color>
     <color>
      <red>128</red>
      <green>128</green>
      <blue>128</blue>
     </color>
     <color>
      <red>141</red>
      <green>139</green>
      <blue>133</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>127</blue>
     </color>
     <color>
      <red>64</red>
      <green>64</green>
      <blue>64</blue>
     </color>
     <color>
      <red>10</red>
      <green>36</green>
      <blue>106</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>232</red>
      <green>232</green>
      <blue>232</blue>
     </color>
    </active>
    <inactive>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>212</red>
      <green>208</green>
      <blue>200</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>234</red>
      <green>229</green>
      <blue>220</blue>
     </color>
     <color>
      <red>128</red>
      <green>128</green>
      <blue>128</blue>
     </color>
     <color>
      <red>141</red>
      <green>139</green>
      <blue>133</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>127</blue>
     </color>
     <color>
      <red>64</red>
      <green>64</green>
      <blue>64</blue>
     </color>
     <color>
      <red>212</red>
      <green>208</green>
      <blue>200</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>232</red>
      <green>232</green>
      <blue>232</blue>
     </color>
    </inactive>
    <disabled>
     <color>
      <red>106</red>
      <green>104</green>
      <blue>100</blue>
     </color>
     <color>
      <red>212</red>
      <green>208</green>
      <blue>200</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>233</red>
      <green>231</green>
      <blue>227</blue>
     </color>
     <color>
      <red>128</red>
      <green>128</green>
      <blue>128</blue>
     </color>
     <color>
      <red>141</red>
      <green>139</green>
      <blue>133</blue>
     </color>
     <color>
      <red>106</red>
      <green>104</green>
      <blue>100</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>106</red>
      <green>104</green>
      <blue>100</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>127</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>127</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>0</blue>
     </color>
     <color>
      <red>10</red>
      <green>36</green>
      <blue>106</blue>
     </color>
     <color>
      <red>255</red>
      <green>255</green>
      <blue>255</blue>
     </color>
     <color>
      <red>0</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>255</red>
      <green>0</green>
      <blue>255</blue>
     </color>
     <color>
      <red>232</red>
      <green>232</green>
      <blue>232</blue>
     </color>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="0" colspan="3" >
      <widget class="QLabel" name="messageLabel" >
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Message&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="actionButton3" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QPushButton" name="actionButton1" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QPushButton" name="closeButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QPushButton" name="actionButton2" >
       <property name="text" >
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3" >
      <widget class="QLabel" name="titleLabel" >
       <property name="text" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:MS Shell Dlg; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Title&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="../wengophone.qrc" >:/emoticons/chat/emoticons/face-kiss48.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../wengophone.qrc" />
 </resources>
 <connections/>
</ui>
